test/client.js

```diff
@@ -251,6 +294,16 @@ test('head with host header', (t) => {
   })
 })
 
+function serverRequestParams (t, expected) {
+  return function (req, res) {
+    const queryParams = querystring.parse(req.url.replace('/?', '')) // We remove opening /? because parser assumes it to be a part of param name
+    t.strictSame(JSON.parse(JSON.stringify(queryParams)), expected)
```

Comment:
For some reason TAP is unhappy here without JSON back-and-forth conversion, because it find a mismatch between `Object` (expected, object literal) and `Null Object` (actual, result of querystring.parse). My googling was unsuccessful here, I have no idea what is going on and how to address it, help from tape gurus would be most welcome here.